src/components/inputs/TextInput.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { Text, TextStyle, View, ViewStyle } from 'react-native';
+import { TextInput as BaseTextInput, TextInputProps } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+
+interface Props extends TextInputProps {
+    placeHolder: string;
+    style?: ViewStyle;
+    onChangeText?: (value: string) => void;
+    value: string;
+    isError?: boolean;
+    renderError?: () => any;
+    disabled?: boolean;
```

Comment:
Hi Vadim, what state this "disabled" option corresponds to regarding the design kit ?

Reply:
Indeed looks like there is no disabled state in the design system for the TextInput element 
<img width="391" alt="Screenshot 2023-03-08 at 2 27 01 PM" src="https://user-images.githubusercontent.com/578618/223725367-7c63ca90-3c4b-4172-8c11-b8c1030bd2aa.png">

Might be an interesting addition though @AudBrou ?

Reply:
Yes it might be, we should check with the PrDs for the design part though